NuGet.Docs/Create/Analyzers-Conventions.md

```diff
@@ -0,0 +1,43 @@
+﻿##Analyzer Nuget Format##
+The proposed format should be identical to the format described [here](https://docs.nuget.org/Create/Enforced-Package-Conventions), except the specifiers in the path describe development host dependencies instead of build-time.
+
+$/analyzers/{**Framework Name**}{**Version**}/{**Supported Architecture**}/**{Supported Programming Language**}/{**Analyzer**}.dll
+
+**Framework Name and Version** the *optional* API surface area of the .NET framework that the contained dlls need to run. If no target is specified, dlls are assumed to apply to *all* targets
```

Comment:
If there is no folder, we assume that the current host can load the dll.  If the user places the dll in a folder whose framework is unsupported or unknown, we skip loading the dll.  If the framework is known and compatible, we load the dll.